src/test/resources/pages/instructorFeedbackCopyTrimmedSuccess.html

```diff
@@ -746,7 +746,7 @@ <h5 class="col-xs-2" data-original-title="Select this option to use a custom tim
   <div aria-hidden="true" aria-labelledby="fsCopyModal" class="modal fade" id="fsCopyModal" role="dialog" tabindex="-1">
     <div class="modal-dialog">
       <div class="modal-content">
-        <form action="/page/instructorFeedbackEditCopy" method="post" name="form_copy_list" role="form">
+        <form action="/page/instructorFeedbackEditCopy" id="instructorCopyModalForm" method="post" role="form">
```

Comment:
I appreciate you trying to make all tests pass, but it remains that these changes shouldn't be part of this PR. Can you revert them?